main/src/main/scala/sbt/internal/ClassLoaders.scala

```diff
@@ -226,6 +226,11 @@ private[sbt] object SbtMetaBuildClassLoader {
     }
     new URLClassLoader(rest, updatedLibraryLoader) {
       override def toString: String = s"SbtMetaBuildClassLoader"
+      override def close(): Unit = {
+        super.close()
+        updatedLibraryLoader.close()
+        interfaceLoader.close()
+      }
```

Comment:
I think `closable.close` may throw `IOException`.

```scala
try {
  super.close()
} finally {
  try {
    updatedLibraryLoader.close(),
  } finally {
     interfaceLoader.close()
  }
}
```

Something like the above is safer?

Reply:
@exoego I think the suggestion is good, but I am ok with merge this as is for now, and following up later. Feel free to send us a PR or open an issue.